wdl-engine/src/value.rs

```diff
@@ -821,7 +1024,7 @@ impl PrimitiveValue {
                 match self.0 {
                     PrimitiveValue::Boolean(v) => write!(f, "{v}"),
                     PrimitiveValue::Integer(v) => write!(f, "{v}"),
-                    PrimitiveValue::Float(v) => write!(f, "{v:?}"),
+                    PrimitiveValue::Float(v) => write!(f, "{v:.6?}"),
```

Comment:
Ah, I see it's intentional here. Could you share the thought process behind being explicit here?